assets/person/tolajoseph.json

```diff
@@ -0,0 +1,20 @@
+{
+    "metadata": {
+        "label": "tolajoseph",
+        "category": "person",
+        "subcategory": "",
+        "organization": "",
+        "description": "",
+        "website": "https://x.com/connectwithtola"
+    },
+    "addresses": [
+        {
+            "address": "UQCWCk-UVSOwYMnrRa5x4Ylx09br2jRE_d6lbwGR0bPw3RkN",
```

Comment:
The address should be in bounceable format EQCWCk-UVSOwYMnrRa5x4Ylx09br2jRE_d6lbwGR0bPw3UTI

put 'individual' in organization 

add couple words in description 

add 'connectwithtola.ton' in comment.